src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -16,12 +18,23 @@ test.describe("Buzz Page", () => {
     await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
   });
 
-  test("Test Case 1: Share file and Delete file", async ({ page }) => {
+  test.afterEach(async ({ page }) => {
+    buzzPage = new BuzzPage(page);
+    await buzzPage.deletePhotos()
+  });
+
+  test("Test 1: Post should be shared", async ({ page }) => {
```

Comment:
OK. And now let's think of this test as a part of a bigger test suite running in parallel... Don't you see the issue with taking the first elements on the list instead of the "correct ones"? I can see at least one good approach that you could take with this.

Reply:
so come on give your approach